.github/Pull_Request_and_Code_Review_Standards.md

```diff
@@ -0,0 +1,45 @@
+# Pull Request and Code Review Standards
+
+-   Please check if there is an existing issue that will be closed by the given PR. Unless it's a trivial change (spelling mistake, code formatting, etc.), this is likely the case.
+
+-   Give the branch for the PR an informative name
+    - If the PR will close an existing issue, name the branch with this format: `\<issue_number\>-short-description` (e.g. \`186_implement_data_extract\`).
```

Comment:
I don't think it's necessary to mention the issue number in the title of the PR because the PR must be linked to the issue in the description anyway. Instead, we could mention [FIX] or [FEATURE] in the title.

Reply:
It doesn't say the title of the PR, but rather the branch from which PR is being made.

Reply:
right, my bad